pages/index.tsx

```diff
@@ -6,6 +6,15 @@ const Home: NextPage = () => {
   return (
     <div className={styles.container}>
       <Head>
+
+      <!-- Google Tag Manager -->
+      <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        })(window,document,'script','dataLayer','GTM-55QBJ7M');</script>
+      <!-- End Google Tag Manager -->
```

Comment:
```suggestion
      {/* End Google Tag Manager */}
```
JSX doesn't allow html comments, you have to comment with {} LGTM!